.github/workflows/sort_draft_prs.yml

```diff
@@ -0,0 +1,55 @@
+name: Move drafts to WIP and un-drafts to seeking reviewer
+on:
+  pull_request_target:
+    types:
+      - opened
+      - converted_to_draft
+      - ready_for_review
+
+env:
+  PROJECT_ID: "PVT_kwDOAKRRkc4AfZil"
+  STATUS_FIELD_ID: "PVTSSF_lADOAKRRkc4AfZilzgUwVMs"
+  WIP_ID: "26e8e6b2"
+  SEEKING_REVIWER_ID: "67583d20"
+  PR_ID: ${{ github.event.pull_request.number }}
+
+jobs:
+  get-token-and-pr-id:
+    name: Get token and pull request ID
+    runs-on: ubuntu-latest
+    outputs:
+      token: ${{ steps.generate-token.outputs.token }}
+      pr-project-id: ${{ steps.get-pr-id.outputs.id }}
+    steps:
+      - name: Generate token
+        id: generate-token
+        uses: actions/create-github-app-token@v1
+        with:
+          app-id: ${{ vars.APP_ID }}
+          private-key: ${{ secrets.APP_PEM }}
+
+      - name: Get PR project ID
+        id: get-pr-id
+        env:
+          GH_TOKEN: ${{ steps.generate-token.outputs.token }}
+        run: |
+          PR_PROJECT_ID="$(gh project item-list --owner Submitty 1 --format json --jq "[.items[] | {id, labels, status, title: .content.title, repo: .content.repository, number: .content.number}] | .[] | select(.number == "$PR_ID").id")"
+          echo "id=$PR_PROJECT_ID" >> "$GITHUB_OUTPUT"
+
+  handle-drafts:
+    name: Move drafts to Work-in-Progress and non-drafts to Seeking Reviewer
+    needs: get-token-and-pr-id
+    env:
+      GITHUB_TOKEN: ${{ needs.get-token-and-pr-id.outputs.token }}
+      PR_PROJECT_ID: ${{ needs.get-token-and-pr-id.outputs.pr-project-id }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Move draft to Work in Progress
+        if: ${{ (github.event.action == 'converted_to_draft') || ((github.event.action == 'opened') && (github.event.pull_request.draft == true)) }}
+        run: |
+          gh project item-edit --project-id "$PROJECT_ID" --id "$PR_PROJECT_ID" --field-id "$STATUS_FIELD_ID" --single-select-option-id "$WIP_ID"
+
+      - name: Move ready for review to Seeking Reviewer
+        if: ${{ (github.event.action == 'ready_for_review') || ((github.event.action == 'opened') && (github.event.pull_request.draft == false)) }}
```

Comment:
I don't think the || part of this is required, as all opened PRs are automatically marked as Seeking (Additional) Reviewer.

Other than that, it looks like it _could_ work, it just needs quite a bit of setup with secrets, and apps, etc.

Reply:
Good point, I can just remove that. The Github App setup has already been completed to my understanding.